### PR TITLE
GET /apps

### PR DESCRIPTION
**Pull request by sh19910711**

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
 ===
 
+`GET /apps`
+-----------
+Returns all applications
+
+**Response:**
+``` json
+[
+    <array of applications>
```
Comment (seiyanuta) [inline API.md]: It should be more specific like:

```json
{
  "applications": [
    {
      "name": "foo"
    }
  ]
}
```

File: API.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
 API
 ===
 
+`GET /apps`
+-----------
+Returns all applications
+
+**Response:**
+``` json
+[
```
Comment (seiyanuta) [inline API.md]: JSON should be Object.

File: app/controllers/apps_controller.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 class AppsController < ApplicationController
+  def index
+    render :json => App.all
```
Comment (seiyanuta) [inline app/controllers/apps_controller.rb]: Use `.plunk` to explicit cloumns.

File: API.md (hunk 1 above)
Comment (sh19910711) [inline API.md]: :+1:

File: app/controllers/apps_controller.rb (hunk 3 above)
Comment (sh19910711) [inline app/controllers/apps_controller.rb]: Nice catch :+1:

**Comment by seiyanuta** [comment]
LGTM 👍

**Comment by sh19910711** [comment]
> LGTM :+1: 

Yay :tada:

Can you merge this pull request? Any time is fine :-)

And thanks for reviewing code.

**Comment by seiyanuta** [comment]
Got it.

**Pull request merged**